BlackMarket/Views/TabBar/MainTabbedView.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  MainTabbedView.swift
+//  BlackMarket
+//
+//  Created by Santiago Andracchi on 26/09/2023.
+//
+
+import SwiftUI
+
+struct MainTabbedView: View {
+  
+  @State var selectedTab = 0
+  
+  var body: some View {
+    
+    ZStack(alignment: .bottom){
+      TabView(selection: $selectedTab) {
+        DashboardView()
+          .tag(0)
+        PurchasesList()
+          .tag(1)
+        ShoppingCartList()
+          .tag(2)
+        FavouriteList()
+          .tag(3)
+        Settings()
+          .tag(4)
+      }
+      
+      ZStack{
+        HStack{
+          ForEach((TabbedItems.allCases), id: \.self){ item in
+            Button{
+              selectedTab = item.rawValue
+            } label: {
+              CustomTabItem(imageName: item.iconName, title: item.title, isActive: (selectedTab == item.rawValue))
+            }
+          }
+        }
+        .padding(6)
+      }
+      .frame(height: 80)
+      .background(.black)
+    }
+  }
+}
+
+extension MainTabbedView {
+  func CustomTabItem(imageName: String, title: String, isActive: Bool) -> some View{
```

Comment:
this seems it should be an independent view instead of a function. If you go for a function, please name it accordingly: 
```suggestion
  func customTabItemView(imageName: String, title: String, isActive: Bool) -> some View {
```